pkgs/development/libraries/mapnik/default.nix

```diff
@@ -1,6 +1,6 @@
 { lib, stdenv, fetchzip
 , boost, cairo, freetype, gdal, harfbuzz, icu, libjpeg, libpng, libtiff
-, libwebp, libxml2, proj, python, sqlite, zlib
+, libwebp, libxml2, proj, python3, python ? python3, sqlite, zlib
```

Comment:
```suggestion
, libwebp, libxml2, proj, python3, sqlite, zlib
```

Reply:
This is to separate the python used as build tool from that for which the binding is created.

Reply:
oh, maybe we should add a comment for this. I am probably not the last one who stumbles upon this.